src/OrbitGl/App.cpp

```diff
@@ -202,6 +203,9 @@ orbit_metrics_uploader::CaptureStartData CreateCaptureStartData(
   capture_start_data.thread_states =
       thread_states ? orbit_metrics_uploader::OrbitCaptureData_ThreadStates_THREAD_STATES_ENABLED
                     : orbit_metrics_uploader::OrbitCaptureData_ThreadStates_THREAD_STATES_DISABLED;
+  // -1 indicates memory collection was turned off
```

Comment:
Instead of a comment here, I prefer to use a constant for the value -1, for instance, `constexpr int64_t kMemoryCollectionDisabled = -1;`

Reply:
done (now its at a different location than before, but I added it as a constant.